README.md

```diff
@@ -0,0 +1,259 @@
+    Successfully allocated Mari Lawrence to Snow
+    Successfully added Leigh Riley to Amity.
+    Successfully added Tana Lopez to Amity.
+    Successfully allocated Tana Lopez to Snow
+    ```
+* `print_allocations [-o FILE]`
+
+    Prints a list of current room allocations onto the screen
+Specifying the optional -o option here outputs the registered allocations to a text 
+file.  
+
+    Below is a sample session where two rooms called Snoe and Oculus exist in the 
+    system:
+    ```
+    (amity >>> ) print_allocations
+    ╒═════════════════╤════════════════╕
+    │ Snoe            │ Oculus         │
+    ╞═════════════════╪════════════════╡
+    │ Simon Patterson │ Oluwafemi Sule │
+    ├─────────────────┼────────────────┤
+    │ Tana Lopez      │ Mari Lawrence  │
+    ╘═════════════════╧════════════════╛
+    ```
+
+* `print_unallocated [-o FILE]`
+
+   Prints a list of unallocated people to the screen.
+Specifying the optional -o option here outputs list of to a text file. 
+
+    Below is a sample session:
+
+    ```
+    (amity >>> ) print_unallocated
+    ╒══════════════════════╕
+    │ Unallocated People   │
+    ╞══════════════════════╡
+    │ Dominic Walters      │
+    ├──────────────────────┤
+    │ Leigh Riley          │
+    ╘══════════════════════╛
+    ```
+* `print_room <room_name>`
+
+   Prints the names of all the people in a room with name `room_name` on the screen.
+   
+   Below is a sample session:
+   ```
+   (amity >>> ) print_room oculus
+    ╒═════════════════════╕
+    │ Oculus occupants:   │
+    ╞═════════════════════╡
+    │ Oluwafemi Sule      │
+    ├─────────────────────┤
+    │ Mari Lawrence       │ 
+    ╘═════════════════════╛
+
+   ```
+
+* `save_state [--db=SQLITE_DATABASE]`
+
+    Persists all the data stored in the app to a SQLite database.
+
+    Specifying the  --db  parameter explicitly stores the data in the 
+    sqlite_database specified. Otherwise, the data is stored in default database
+    called `amity_db`
+    
+    Below is a sample session:
+    ```
+    (amity >>> ) save_state --db store_db
+    Successfully saved Amity state to  store_db sqlite database
+    ```
+* `load_state <sqlite_database>`
+
+    Loads application state from the specified database into 
+    the application.
+    
+    The database file provided must be a valid sqlite3 database.
+    
+    Below is a sample session:
+    
+    ```
+    (amity >>> ) load_state store_db
+    Successfully loaded state from store_db
+    ```
+* `quit`
+
+    This exits the application.
+
+* `list_rooms`
+
+    Prints the all rooms in the system on the screen.
+    
+    Below is a sample session:
+    
+    ```
+    (amity >>> ) list_rooms
+    ╒═══════════════╕
+    │ Amity Rooms   │
+    ╞═══════════════╡
+    │ Snoe          │
+    ├───────────────┤
+    │ Oculus        │
+    ╘═══════════════╛
+    ```
+    
+See the video below to get a compelete picture of how to use Office
+Room Allocator:
+
+
```

Comment:
Nice Work on the readMe it is pretty detailed and structured, also make sure to merge all your changes to master at the end since `git clone https://github.com/andela-mkamau/Office-Space-Allocation.git` clones an empty repo currently.